src/commands/general.ts

```diff
@@ -0,0 +1,66 @@
+import { select, text } from "@clack/prompts";
+import { promptWithCancel } from "./promptWithCancel";
+import { config, saveConfig } from "../utils/cli.utils";
+
+const general = async (options: {
+  backupLocation?: string;
+  logLocation?: string;
+  logLevel?: string;
+  retentionPolicy?: string;
+  backupSchedule?: string;
+}) => {
+  if (!Object.keys(options).length) {
+    const backupLocation = await promptWithCancel(text, {
+      message: "Enter backup location",
+      initialValue: config.general.backup_location,
+    });
+
+    const logLocation = await promptWithCancel(text, {
+      message: "Enter log location",
+      initialValue: config.general.log_location,
+    });
+
+    const logLevel = await promptWithCancel(select, {
+      message: "Select log level",
+      options: [
+        { value: "INFO", label: "INFO" },
+        { value: "DEBUG", label: "DEBUG" },
+        { value: "ERROR", label: "ERROR" },
+      ],
+      initialValue: config.general.log_level,
+    });
+
+    const retentionPolicy = (await promptWithCancel(text, {
+      message: "Enter retention policy (days)",
+      initialValue: String(config.general.retention_policy_days),
+    })) as unknown as number;
```

Comment:
@theanupambista changing type of retention_policy_days is better than using as unknown as number?
since you are converting value to Number.

Reply:
@mukezhz You're correct that converting the value to a number directly is a cleaner approach compared to using as unknown as number. 
I will fix this in next commit